### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yangyang480**
to here :)

home you approve 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,4 @@ mountaintechnologiesllc.com
 
 This app was originally built using command: `dotnet new angular -n mt-app -o mountain-technologies-llc`
 
+Yangyang was here :)
```
Comment (ajgoldenwings) [inline README.md:6]: Really?

**Review by ajgoldenwings — CHANGES_REQUESTED** [review]
Please respond to comment...

File: README.md (hunk 1 above)
Comment (yangyang480) [reply README.md:6]: YUP :) :P 🥇

Comment (ajgoldenwings) [reply README.md:6]: Ok [facepalm]

**Comment by ajgoldenwings** [comment]
lgtm

**Review by ajgoldenwings — APPROVED** [review]
lgtm

**Pull request merged**